evergreen/run-tests.sh

```diff
@@ -25,9 +25,9 @@ provision_ssl () {
   uri_environment_variable_name=$1
   # Arguments for auth + SSL
   if [ "$AUTH" != "noauth" ] || [ "$TOPOLOGY" == "replica_set" ]; then
-    export $uri_environment_variable_name="${!uri_environment_variable_name}&ssl=true&sslVerifyCertificate=false"
+    export $uri_environment_variable_name="${!uri_environment_variable_name}&ssl=true"
```

Comment:
We no longer need `sslVerifyCertificate=false` since we're adding the issuing ca to the system trust store